DevelopGitHub/GitHubExtension/GitHubExtension.DB/Templates.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[Templates]
+(
+	[Id] INT NOT NULL IDENTITY(1,1),
+	[Content] NVARCHAR (MAX) NOT NULL,
+	[Type] NVARCHAR(50) NOT NULL, 
+    [CategoryId] INT NULL,
```

Comment:
It's very strange)
In VS and NotePad++ normal alignment.
<img width="312" alt="2016-04-27 10 22 35" src="https://cloud.githubusercontent.com/assets/10244166/14844701/f9cbc952-0c61-11e6-8293-5dcd9a736a52.png">
<img width="647" alt="2016-04-27 10 19 56" src="https://cloud.githubusercontent.com/assets/10244166/14844702/fb42d064-0c61-11e6-93dc-9a16b8386944.png">